src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -22,25 +22,38 @@ clusters with overlapping CIDRs to connect together.
 
 To support overlapping CIDRs in connected clusters, Submariner has a component called Global Private Network, Globalnet (`globalnet`). This
 Globalnet is a virtual network specifically to support Submariner's multi-cluster solution with a global CIDR. Each cluster is given a
-subnet from this virtual Global Private Network, configured as new cluster parameter `GlobalCIDR` (e.g. 169.254.0.0/16) which is
+subnet from this virtual Global Private Network, configured as new cluster parameter `GlobalCIDR` (e.g. 242.0.0.0/8) which is
 configurable at time of deployment. User can also manually specify GlobalCIDR for each cluster that is joined to the Broker using the flag
 ```globalnet-cidr``` passed to ```subctl join``` command. If Globalnet is not enabled in the Broker or if a GlobalCIDR is preconfigured in
 the cluster, the supplied globalnet-cidr will be ignored.
 
-Once configured, each exported Service and Pod that requires cross-cluster access is allocated an IP, named `globalIp`,
-from this `GlobalCIDR` that is annotated on the Pod/Service object.
-This globalIp is used for all cross-cluster communication to and from a Pod and the globalIp of a
-remote Service. Routing and iptable rules are configured to use the globalIp for ingress and egress. All address translations occur on the
-Gateway node.
+### ClusterGlobalEgressIPs
 
-![Figure 1 - Proposed solution](/images/globalnet/overlappingcidr-solution.png)
+By default every cluster is assigned configurable number of GlobalIPs which is used as egress IP for any cross cluster communication.
+Multiple IPs are supported to avoid ephemeral port exhaustion issues. Default is `8`.
+Applications on HostNetwork trying to access remote services will also use ClusterGlobalEgressIPs.
 
-{{% notice info %}}
+### GlobalEgressIPs
 
-Unlike vanilla Submariner, where Pod to Pod connectivity is also supported, Globalnet only supports Pod to remote Service connectivity using
-globalIps.
+User can create GlobalEgressIPs at scope namespace which take precedence over ClusterGlobalEgressIPs, if configured. All pods in the
+namespace will use GlobalEgressIP allocated for that namespace.
 
-{{% /notice %}}
+User can also configure GlobalEgressIPs for a specific set of pods in a namespace by using a `PodSelector` when creating GlobalEgressIP.
+These GlobalEgressIPs will take precedence over any globalIps allocated at namespace or cluster level.
```

Comment:
```suggestion
These GlobalEgressIPs will take precedence over any globalIPs allocated at namespace or cluster level.
```

I think IP should always be fully capitalized, right? (applies elsewhere).

Reply:
This paragraph isn't needed with my suggestion above.